#Expyriment 0.9.0 (Python 3.6.6), .xpe-file, coding: ISO8859-1
#date: Tue Aug 14 2018 15:30:31
#sha1: None
#modules: 
#display: size=(800, 600), window_mode=True, open_gl=2
#os: uname_result(system='Linux', node='L3460', release='4.4.0-109-generic', version='#132~14.04.1-Ubuntu SMP Tue Jan 9 21:46:42 UTC 2018', machine='x86_64', processor='x86_64')
Time,Type,Event,Value,Detail,Detail2
325,stdout,received,'LEVEL 1'
326,stderr,received,'Traceback (most recent call last):'
326,stderr,received,'  File "arithmatictask_v0.py", line 47, in <module>'
326,stderr,received,'    b.add_trial(stimuli.Stimuli)'
326,stderr,received,"AttributeError: module 'expyriment.stimuli' has no attribute 'Stimuli'"
